airbyte-webapp/src/views/Connector/ServiceForm/ServiceForm.test.tsx

```diff
@@ -244,7 +244,9 @@ describe("Service Form", () => {
         <ServiceForm
           formType="source"
           formValues={{ name: "test-name", serviceType: "test-service-type" }}
-          onSubmit={(values) => (result = values)}
+          onSubmit={(values) => {
+            result = values;
+          }}
```

Comment:
Not sure we needed to make this change but :shrug:

Reply:
Typing, since `(result = values)` will return the result of that asignment, i.e. the asigned value. Since I typed it as `Promise<void>` though this method should not return anything thus switching to a block level method. 

In theory the method could have been typed as `Promise<unknown>` to say we simply don't care about the return type, but imho that sets us up for slightly more dangerous path again that at some point somenoe starts using that return value again. Since it was just one case and this test that didn't return Promise<void>, I decided to go with Promise<void> over Promise<unknown>